tasks/affective_task/server_affective_task.py

```diff
@@ -79,8 +82,6 @@ def run(self, images_dir: str, collaboration: bool = False):
 
             while(True):
                 monotonic_time = monotonic()
```

Comment:
why do you capture monotonic time here? It will become invalid after the `receive` function because the `receive` method freezes the software until it gets the data. Just use `"monotonic_time": monotonic()` in the `writerow` method.

Reply:
Done